src/Apps/Collect2/Routes/Collect/Components/CollectMediumMetadata.ts

```diff
@@ -11,7 +11,7 @@ export function getMetadataForMedium(medium) {
       title = "Photography"
       mediumDescription = "140,000 photographs"
       break
-    case "sculptures":
+    case "sculpture":
```

Comment:
Why is this one changing? I wonder if we should leave both, there are links to this on the homepage.

Reply:
This one is currently broken: 
![image](https://user-images.githubusercontent.com/2081340/68884789-e2297f00-06e1-11ea-879d-7a291585e8c5.png)

So this should fix the issue but not create a new one.

Reply:
(I got to that by clicking from the homepage, btw!)

Reply:
Cool just making sure we weren't killing an existing link!